docs/resources/Channel.md

```diff
@@ -700,6 +705,8 @@ This endpoint supports requests with `Content-Type`s of both `application/json`
 > info
 > Note that when sending `application/json` you must send at least one of `content` or `embed`, and when sending `multipart/form-data`, you must send at least one of `content`, `embed` or `file`. For a `file` attachment, the `Content-Disposition` subpart header MUST contain a `filename` parameter.
 
+You may create a message as a reply to another message. To do so, include a [`message_reference`](#DOCS_RESOURCES_CHANNEL/message-object-message-reference-structure) with a `channel_id` and `message_id`. This requires the `VIEW MESSAGE HISTORY` permission, the reference message cannot be a system message, and the channel id of the referenced message must match the channel that you're sending the message in.
```

Comment:
Is `channel_id` really needed here? We're already specifying which channel messages are sent to, having a duplicate reference doesn't seem to make sense here.

Reply:
I think the concept is that you can inline-reply to any message maybe? Like looking at the comment below showing how it looks client side, it doesn't seem to be replying directly to the message, rather creating a separate message mentioning that message. Maybe not in the client (just cuz it seems kinda impractical), but in the API, might be able to send reply's to a message in a different channel than that of which that message is from.

Reply:
it would be useful (and was technically possible in replies v1 by typing out the syntax), but in the line it says
>  the channel id of the referenced message must match the channel that you're sending the message in

hopefully it gets changed tho

Reply:
To be fair in logic if someone (for example) DMs you on Discord, you wouldn't reply to them by text (SMS), that would just be weird. I'd probably expect the same here but with channels. In which case if that is true then channel_id definitely won't be needed

Reply:
The Discord client with the inline replies experiment enabled can reply to any message from any channel without the restriction of the channel being the same.

Reply:
Well this is both true and not. In DMs yes, however being able to reply to something in a different channel in the server the message is in, doesn't come off as unworldly. It'd be neat to see, maybe even for like logging purposes? Being able to reply to the message in a log channel per say to keep track of it. Or replying to like an announcement in a general chat for users. It doesn't seem like something that shouldn't happen, but still might not.

Reply:
This would be useful for, say, moving a conversation to another channel.

Reply:
This has been changed in the latest update--only `message_id` is required.